### PR TITLE
fix: refactor get page

### PR DESCRIPTION
**Pull request by sammcj**
- fix: refactor get page using the newly included GraphQL type with GraphQlQueryResponseData

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   ],
   "dependencies": {
     "@octokit/core": "^3.5.1",
+    "@octokit/graphql": "^5.0.1",
```
Comment (domdomegg) [inline package.json]: I think you can install this as a devDependency instead as we're just using the type from it.

File: src/index.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 import _prompts from 'prompts';
 import { Octokit } from "@octokit/core";
+import { GraphQlQueryResponseData } from '@octokit/graphql';
```
Comment (domdomegg) [inline src/index.ts]:
```suggestion
import type { GraphQlQueryResponseData } from '@octokit/graphql';
```

**Comment by domdomegg** [comment]
I might be missing something here, but is this primarily reformatting? I don't spot any logic/wider structural changes.

Happy to approve (after #2), just want to check I've understood it correctly.

**Comment by sammcj** [comment]
Yeah some formatting - but that's around where I induced the missing GraphQL type (GraphQlQueryResponseData) which sorted out a bunch of warnings.

**Comment by domdomegg** [comment]
I see you've updated this PR from a draft to ready for review. Leaving this comment to flag I'm assuming this means 'ready for review pending #2, which is in a draft status' and thus I'm gonna wait on that one.

Please let me know if I'm incorrect here! :)

**Comment by sammcj** [comment]
Hey sorry @domdomegg I've become super busy all of the sudden! This PR can be merged at any time if you're happy with it, it doesn't actually depend on #2 which I might even cancel and start fresh at some point.

**Comment by domdomegg** [comment]
I think this PR still contains some of the same issues as raised on #2 in that case - can you address those before merging? (e.g. https://github.com/domdomegg/github-repo-settings-editor/pull/2#discussion_r987422519 https://github.com/domdomegg/github-repo-settings-editor/pull/2#discussion_r987423405 https://github.com/domdomegg/github-repo-settings-editor/pull/2#discussion_r987424588 https://github.com/domdomegg/github-repo-settings-editor/pull/2#discussion_r987426614)

**Pull request closed without merging**